includes/Modules/Idea_Hub.php

```diff
@@ -146,12 +161,126 @@ function( $maybe_empty, $postarr ) {
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $notices Array of admin notices.
+	 * @return array Array of admin notices.
+	 */
+	private function admin_notice_idea_hub_ideas( $notices ) {
+		global $post_type;
+		$current_screen = get_current_screen();
+		if ( is_null( $current_screen ) || 'edit-post' !== $current_screen->id || 'post' !== $post_type ) {
+			return $notices;
+		}
+		$transients      = new Transients( $this->context );
+		$dismissed_items = new Dismissed_Items( $this->user_options );
+
+		$notices[] = new Notice(
+			'idea-hub_saved-ideas',
+			array(
+				'content'         => function() {
+					$saved_notice = array(
+						'text' => esc_html__( 'Need some inspiration? Revisit your saved ideas in Site Kit', 'google-site-kit' ),
+						'cta'  => esc_html__( 'See saved ideas', 'google-site-kit' ),
+						'link' => $this->context->admin_url() . '#new-ideas',
+					);
+					ob_start();
+					?>
+					<p>
+						<?php echo esc_html( $saved_notice['text'] ); ?>
+						<a href="<?php echo esc_url( $saved_notice['link'] ); ?>"
+						><?php echo esc_html( $saved_notice['cta'] ); ?></a>
+					</p>
+					<?php
+					return ob_get_clean();
+				},
+				'type'            => Notice::TYPE_INFO,
+				'active_callback' => function() use ( $transients, $dismissed_items ) {
+					$saved_ideas = $transients->get( self::TRANSIENT_SAVED_IDEAS );
+					if ( false === $saved_ideas ) {
+						$saved_ideas = $this->get_data( 'saved-ideas' );
+						$transients->set( self::TRANSIENT_SAVED_IDEAS, $saved_ideas, DAY_IN_SECONDS );
+					}
+					$has_saved_ideas = count( $saved_ideas ) > 0;
+					if ( ! $has_saved_ideas && $dismissed_items->is_dismissed( 'saved-ideas' ) ) {
+						// Saved items no longer need to be dismissed as there are none currently.
+						$dismissed_items->delete( 'saved-ideas' );
+					}
+					if ( $dismissed_items->is_dismissed( 'saved-ideas' ) ) {
+						return false;
+					}
+
+					return $has_saved_ideas && ! $dismissed_items->is_dismissed( 'saved-ideas' );
+				},
+				'dismissible'     => true,
+			)
+		);
+		$notices[] = new Notice(
+			'idea-hub_new-ideas',
+			array(
+				'content'         => function() {
+					$new_notice = array(
+						'text' => esc_html__( 'Need some inspiration? Here are some new ideas from Site Kit’s Idea Hub', 'google-site-kit' ),
+						'cta'  => esc_html__( 'See new ideas', 'google-site-kit' ),
+						'link' => $this->context->admin_url() . '#saved-ideas',
+					);
+					ob_start();
+					?>
+					<p>
+						<?php echo esc_html( $new_notice['text'] ); ?>
+						<a href="<?php echo esc_url( $new_notice['link'] ); ?>"
+						><?php echo esc_html( $new_notice['cta'] ); ?></a>
+					</p>
+					<?php
+					return ob_get_clean();
+				},
+				'type'            => Notice::TYPE_INFO,
+				'active_callback' => function() use ( $transients, $dismissed_items ) {
+					if ( $dismissed_items->is_dismissed( 'new-ideas' ) ) {
+						return false;
+					}
```

Comment:
We also need to check whether the `saved-ideas` notice is dismissed too. If either of these notices is dismissed, we should return early.

Reply:
I don't _think_ we need to check for saved-ideas because if `saved-ideas` is dismissed, you can still get a notice for new ideas, no?

Isn't the check to avoid double notifying?

Reply:
Updated in https://github.com/google/site-kit-wp/pull/3668/commits/4a39e97f48c93a1ed4800eb5e93e15bdbbeadc6a